### DISCUSSION TITLE
3.4.0

### DISCUSSION THREAD

**Discussion by quantumsheep** [quantumsheep/sshs#46] [Announcements]
## What's Changed
* Removed Nix advisory (Backport PR Merged) by @not-my-segfault in https://github.com/quantumsheep/sshs/pull/38
* Add option to exit instead of returning to sshs by @nodauf in https://github.com/quantumsheep/sshs/pull/45
* Allows wraps vertically by @nodauf in https://github.com/quantumsheep/sshs/pull/44

## New Contributors
* @nodauf made their first contribution in https://github.com/quantumsheep/sshs/pull/45

**Full Changelog**: https://github.com/quantumsheep/sshs/compare/3.3.0...3.4.0

<hr /><em>This discussion was created from the release <a href='https://github.com/quantumsheep/sshs/releases/tag/3.4.0'>3.4.0</a>.</em>

**Comment by mchalunderscore** [comment]
Already PRed against `nixpkgs`! :smile: 

https://github.com/NixOS/nixpkgs/pull/199921

**Comment by mchalunderscore** [comment]
I would also gladly update it on the AUR if possible, my username on the AUR is `mchal`!

**Reply by quantumsheep** [reply]
You are now co-maintainer :) https://aur.archlinux.org/packages/sshs

**Reply by mchalunderscore** [reply]
Bumped version on the AUR now too! :smile:

**Reply by quantumsheep** [reply]
Thanks a lot helping the project!

**Reply by mchalunderscore** [reply]
I use `sshs` personally, it's the least I can do! :grin:
